object/src/meta.rs

```diff
@@ -372,7 +345,10 @@ impl FileMetaTableBuilder {
             .ok_or_else(|| Error::InvalidFormat)?;
         let information_version = self
             .information_version
-            .ok_or_else(|| Error::InvalidFormat)?;
+            .or_else(|| {
```

Comment:
`unwrap_or_else` may be handy here

Reply:
Indeed, thanks! For some reason my IDE did not suggest me that method.